src/style.css

```diff
@@ -0,0 +1,55 @@
+html,
+body {
```

Comment:
set styles for some special tags, elemets

eg `html` doesn't have margin ,padding, so no need to reset it

why do you need width, height 0?